.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,7 +27,7 @@ N/A
 
 ### Submitter checklist
 
-- [ ] JIRA issue is well described
+- [ ] Jira issue is well described (not mandatory)
```

Comment:
Ambiguous:

* Is it not necessary to have a Jira issue?
* Is it not necessary that the Jira issue is useful?

Reply:
What if I adjust it like...

```suggestion
- [ ] Jira issue is well described (if needed)
```

Can just revert it tho

Reply:
How about

```suggestion
- [ ] (If applicable) Jira issue is well described
```